client/python/openlineage/client/transport/kafka.py

```diff
@@ -56,4 +56,4 @@ def __init__(self, config: KafkaConfig):
     def emit(self, event: RunEvent):
         self.producer.produce(topic=self.topic, value=Serde.to_json(event).encode('utf-8'))
         if self.flush:
-            self.producer.flush(timeout=2)
+            self.producer.flush(timeout=5)
```

Comment:
Wouldn't it be nice to let user override the timeout? 
If so, consider creating a separate issue for that.